collectors/apicmcpfault.py

```diff
@@ -0,0 +1,58 @@
+import logging, re
+
+from prometheus_client.core import CounterMetricFamily, Summary
+import BaseCollector
+
+LOG = logging.getLogger('apic_exporter.exporter')
+REQUEST_TIME = Summary('network_apic_mcp_fault_counter',
+                       'Time spent processing request')
+
+
+class ApicMCPCollector(BaseCollector.BaseCollector):
+    def describe(self):
+        yield CounterMetricFamily('network_apic_mcp_fault_counter',
+                                  'Counter for MCP Faults')
+
+    @REQUEST_TIME.time()
+    def collect(self):
+        LOG.debug('Collecting APIC MCP Fault metrics ...')
+
+        g_dip = CounterMetricFamily(
```

Comment:
As this is counter for mcp faults, I suggest to rename variable _g_dip_ to _c_mcp_faults_ in the code.

Reply:
updated